apiserver/facades/client/cloud/cloud.go

```diff
@@ -794,6 +794,15 @@ func (api *CloudAPI) ModifyCloudAccess(ctx context.Context, args params.ModifyCl
 			result.Results[i].Error = apiservererrors.ServerError(err)
 			continue
 		}
+		if !api.isAdmin {
```

Comment:
issue: The facade now needs to check that the apiUserName has permission sufficient to change the permissions. They need SuperUser across the controller or Admin for the entity.

isAdmin checks for SuperUser